main.js

```diff
@@ -0,0 +1,127 @@
+let key = openWeather.key;
+let weatherCalls = [];
+let unit = 'imperial';
+
+let search = {
+  btn: document.getElementById('searchBtn'),
+  input: document.getElementById('searchInput'),
+}
+
+let dailyWeatherDiv = document.getElementById('dailyWeather');
+let dailyWeatherIconDiv = document.getElementById('dailyWeatherIcon');
+let forcastDiv = document.getElementById('forcastDiv');
+
+search.btn.addEventListener('click', () => {
+  let inputElement = document.getElementById('searchInput');
+  let city = inputElement.value
+  console.log('Button Pressed')
+
+  getWeather(city);
+
+  inputElement.value = "";
+  forcastDiv.innerHTML = "";
+
+
+})
+
+search.input.addEventListener('keypress', (e) => {
+  let key = e.key
+  console.log(key);
+
+  if (key === 'Enter') {
+    search.btn.click()
+    search.input.blur()
+  }
+})
+
+var getWeather = function (city = "Clermont") {
```

Comment:
switching back to pre-es6 variable declarations is not recommend, stay consistent with your code